parent/pom.xml

```diff
@@ -16,13 +16,14 @@
     <url>http://microsoft.com</url>
   </organization>
 
+<!--
   <parent>
```

Comment:
If we decide to stick without this we should just remove it instead of commenting it out.

Reply:
Agreed. I'll push another PR where I remove this.

Reply:
Done. Latest commit has this removed.